addons/board/models/board.py

```diff
@@ -1,14 +1,17 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class Board(models.AbstractModel):
     _name = 'board.board'
     _description = "Board"
     _auto = False
 
+    # this is necessary for some nonsensical onchange...
+    id = fields.Id()
```

Comment:
wut? which one? can't seem to find much about this model

Reply:
Well, some test was failing.
For some reason, the web client performs an `onchange()` on this model.  I don't know why.
But I added the field, and it now works :wink:

Reply:
I found out the explanation: the module "board" defines "My dashboard", which uses the old technology for dashboards, i.e., a special form view on model `board.board`.  When you open "My dashboard", technically speaking the client opens a form view, which initializes a record (hence the call `onchange()`)... for nothing.

Reply:
Not great but nothing we can really do, this will do 👍🏽